src/Foundation/InstalledApp.php

```diff
@@ -48,7 +48,7 @@ public function __construct(Container $container, array $config)
     public function getRequestHandler()
     {
         if ($this->inMaintenanceMode()) {
-            return new MaintenanceModeHandler();
+//            return new MaintenanceModeHandler();
```

Comment:
Not sure why I commented this out, but it should probably go, including the handler.